src/views/packagings/PackagingTable.tsx

```diff
@@ -37,6 +40,11 @@ const extractRowValues = (packaging: ModelPackaging) => [
 ];
 
 export const PackagingTable: React.FC = () => {
+
+    const deployableIntegrations = ["docker-rest"]
```

Comment:
Looks like eventually we should add `deployable` field to `Deployment` entity on server side.

Reply:
Agree, it looks consistent. But It seems that each new deployable integration requires a specific UI view for itself. So there is no way to enable deployment feature on UI for the integration by just some flag from the configuration. It anyway UI requires code changes. That is why I decided just to add const to UI code for now.